### DISCUSSION TITLE
PyTorch-Ignite in GitHub Trending

### DISCUSSION THREAD

**Discussion by vfdev-5** [pytorch/ignite#1730] [General]
<img width="512" alt="Screen Shot 2021-03-02 at 17 37 46" src="https://user-images.githubusercontent.com/2459423/109715111-85d2c300-7ba3-11eb-908a-952bc09f8803.png">

Probably, that could explain "71 stars today".

**Comment by sdesrozis** [comment]
Good, isn't it ? 😀
